metadataCreator/src/utils/markdownUtils.ts

```diff
@@ -35,19 +35,39 @@ export function markdownMultisigTable(multisigData: MultisigVersionStorage) {
     return markdownTable
 }
 
+export function markdownProxyTable(chains: Chain[]) {
+    let markdownTable = `
+# List of Networks where we are support proxy feature
+| -- | Network | Pure proxy | Regular proxy |
+| -------- | -------- | -------- | -------- |
+`
+    let counter = 0;
+    chains.forEach(chain => {
+        const { networkName, pure, regular } = calculateProxyDataForTable(chain);
+        if (pure || regular) {
+            counter++;
+            markdownTable += `| ${counter} | ${networkName} | ${pure} | ${regular} |\n`;
+        }
+    });
```

Comment:
reduce looks better here 

```suggestion
    const counter chains.reduce((acc, chain) => {
        const { networkName, pure, regular } = calculateProxyDataForTable(chain);
        if (pure || regular) {
            acc++;
            markdownTable += `| ${counter} | ${networkName} | ${pure} | ${regular} |\n`;
        }
        
        return acc
    }, 0);
```